### PR TITLE
Migrate to the harvester installer

### PR DESCRIPTION
**Pull request by guangbochen**
1. migrate backend code to the [harvester-server](https://github.com/rancher/harvester-server)
1. Add ISO build code based on [harvester-installer](https://github.com/rancher/harvester-installer)

### CODE REVIEW

File: Dockerfile.dapper [hunk 1]
```diff
@@ -1,44 +1,35 @@
...
+ENV HELM_URL=https://get.helm.sh/helm-${HELM_VERSION}-linux-${ARCH}.tar.gz
+RUN mkdir /usr/tmp && \
+    curl ${HELM_URL} | tar xvzf - --strip-components=1 -C /usr/tmp/ && \
+    mv /usr/tmp/helm /usr/bin/helm
 
-# -- for dapper
 ENV GO111MODULE off
 ENV DAPPER_ENV REPO TAG DRONE_TAG CROSS
-ENV DAPPER_SOURCE /go/src/github.com/rancher/harvester/
+ENV DAPPER_SOURCE /go/src/github.com/rancher/harvester-installer/
```
Comment (gitlawr) [inline Dockerfile.dapper]: Keep the original

File: go.mod [hunk 2]
```diff
@@ -1,51 +1,29 @@
-module github.com/rancher/harvester
+module github.com/rancher/harvester-installer
```
Comment (gitlawr) [inline go.mod]: Update the module name

File: pkg/console/console.go [hunk 3]
```diff
@@ -0,0 +1,123 @@
+package console
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/jroimartin/gocui"
+	"github.com/rancher/harvester-installer/pkg/widgets"
```
Comment (gitlawr) [inline pkg/console/console.go]: Looks like the import paths are not updated

**Comment by guangbochen** [comment]
close in favor of the migration process is deprecated.

**Pull request closed without merging**